compiler/rustc_parse/src/parser/expr.rs

```diff
@@ -1440,21 +1440,23 @@ impl<'a> Parser<'a> {
             } else if this.eat_keyword(kw::Underscore) {
                 Ok(this.mk_expr(this.prev_token.span, ExprKind::Underscore))
             } else if this.token.uninterpolated_span().at_least_rust_2018() {
-                // `Span:.at_least_rust_2018()` is somewhat expensive; don't get it repeatedly.
-                if this.check_keyword(kw::Async) {
+                // `Span::at_least_rust_2018()` is somewhat expensive; don't get it repeatedly.
```

Comment:
```suggestion
                // `Span::at_least_rust_2024()` is somewhat expensive; don't get it repeatedly.
```

Reply:
That comment actually applies to the line above it, not below.

Reply:
It also applies to the line below it, which is why it's written like that... but it was originally here before any of the `gen` stuff was written :)

Reply:
Ah, right!